f8a_report/helpers/report_helper.py

```diff
@@ -90,6 +90,56 @@ def cleanup_db_tables(self):
             # Log the message returned from db cursor
             logger.info('%r' % self.cursor.statusmessage)
             logger.info('Cleanup of Worker Result data tables complete')
+
+            # Number of days to retain the package_analyses data
+            num_days_package_analyses = os.environ.get('KEEP_PACKAGE_ANALYSES_NUM_DAYS', '30')
+            # query to delete the worker_result data
+            query = sql.SQL('DELETE FROM package_analyses '
+                            'WHERE ended_at <= NOW() - interval \'%s day\';')
+            logger.info('Starting to clean up Package Analyses data tables')
+            # Execute the query
+            self.cursor.execute(query.as_string(self.conn) % (num_days_package_analyses))
+            # Log the message returned from db cursor
+            logger.info('%r' % self.cursor.statusmessage)
+            logger.info('Cleanup of Package Analyses data tables complete')
+
```

Comment:
Can you refactor these into individual functions? One per each type of table purge.

Reply:
Yes @arajkumar on it.